README.md

```diff
@@ -688,6 +689,14 @@ Type: _string_
 
 Default: `""`
 
+#### `domain_website`
+
+Set this to the domain that you want to deploy your website to. By default, it will be deployed to a `PROJECT_NAME.pages.dev` Cloudflare Pages URL.
```

Comment:
@anujdeshpande Having two inputs makes little sense. Can we not use the same parameter for both? (i.e if the parameter doesn't include `.` dots, then assume it's `${{ website }}.pages.dev`, otherwise assume a FQDN.

Reply:
@ab77 That would add complexity to the action as we would have to figure out the pages project name from the URL. We use the official Cloudflare action which uses their wrangler CLI - and that only allows one to do `wrangler pages publish --project-name $PROJECT_NAME`

https://developers.cloudflare.com/workers/wrangler/commands/#publish-1

So basically flowzone/docusaurus action would have to 
- figure out it's a FQDN
- (using cURL)list all the pages project and their domains 
- find the project name from that list

Or we have a lookup table somewhere (but that's more complicated than this)

Reply:
@anujdeshpande you can just run a bash step before where you do all the figuring out and provide and formatted output for the next step to consume.